dataland-frontend/src/components/resources/frameworkDataSearch/DataModelsTranslations.ts

```diff
@@ -0,0 +1,374 @@
+export const lksgKpis = {
+  dataDate: "Data Date",
+  companyLegalForm: "Company Legal Form",
+  vatIdentificationNumber: "VAT Identification Number",
+  numberOfEmployees: "Number Of Employees",
+  shareOfTemporaryWorkers: "Share Of Temporary Workers",
+  totalRevenue: "Total Revenue",
+  totalRevenueCurrency: "Total Revenue Currency",
+  responsibilitiesForFairWorkingConditions: "Responsibilities For Fair Working Conditions",
+  responsibilitiesForOccupationalSafety: "Responsibilities For Occupational Safety",
+  riskManagementSystem: "Risk Management System",
+  betterWorkProgramCertificate: "Better Work Program Certificate",
+  grievanceHandlingMechanismUsedForReporting: "Grievance Handling Mechanism Used For Reporting",
```

Comment:
Dropdown in Company Search: Schreibweise LkSG statt Lksg

Reply:
fixed

Reply:
fixed